src/main/webapp/dataverse_header.xhtml

```diff
@@ -189,6 +211,11 @@
                 </div>
             </div>
         </noscript>
+        <script>
+function a () {
+var contact = prompt("Add the contact.");
+}
```

Comment:
Is this code just for debugging?

Reply:
Yes. I have removed it

Reply:
Whoops. Sorry. I didn't mean to include that in my review. I saw you had already deleted it. I forgot to go back and delete the comment.